PSReadLine/Cmdlets.cs

```diff
@@ -309,6 +333,16 @@ public object ContinuationPromptColor
         public string HistorySavePath { get; set; }
         public HistorySaveStyle HistorySaveStyle { get; set; }
 
+        /// <summary>
+        /// Sets the source to get predictive suggestions.
+        /// </summary>
+        public PredictionSource PredictionSource { get; set; }
+
+        /// <summary>
+        /// How the predictive suggestion is rendered.
+        /// </summary>
+        public PredictionViewStyle PredictionViewStyle { get; set; }
```

Comment:
Can we remove this until there is a need?

Reply:
Will do.